docker/Dockerfile-rabbitmq

```diff
@@ -0,0 +1,4 @@
+FROM rabbitmq:3.7-management
```

Comment:
I'm looking at https://hub.docker.com/_/rabbitmq, and it seems that the `-management` variants should already have management enabled by default (search for "Management Plugin") :thinking:

Reply:
Thank you, yes, I can see it in rabbitmq-management [Dockerfile](https://github.com/docker-library/rabbitmq/blob/7e63843da6bfb191ddee6dbe3dd7ec0df36ae70b/3.8-rc/ubuntu/management/Dockerfile#L9). 
But without this changes I still getting this error: 
```
flower_1          | [E 210201 13:45:01 broker:78] RabbitMQ management API call failed: Timeout while connecting
```
Could you please check it locally? 
I am trying to find a solution without double-enabling it.

Reply:
I reverted the image back to `rabbitmq:3.7-management` while keeping the Flower command the same
```diff
-    build:
-      context: .
-      dockerfile: ./docker/Dockerfile-rabbitmq
+    image: rabbitmq:3.7-management
```
and I'm no longer seeing this error. I think the `--broker_url=` option fixed it.

But there is another slight problem. If I run `make run`, i.e. deploy everything at the same time, I still do not see any output in http://localhost:8083/broker. If I then restart Flower (`docker-compose restart flower`), things start to work. I think we want to wait for rabbitmq to be up before starting flower.

Reply:
> I think we want to wait for rabbitmq to be up before starting flower.

Hmm, that's not it, I added a 30s sleep and it still only worked after restart (rabbitmq was definitely up by then). Needs more investigation.

Reply:
@akhmanova , any updates on this one? Could you try a longer sleep as Adam suggested?

Also, see https://docs.docker.com/compose/startup-order/

Reply:
@chmeliik Thank you for the investigation! 
@athos-ribeiro Yes, thank you. I added a file `flower-waits-for-rabbitmq.sh` to check the start of Rabbitmq. 
Looks like docker compose doesn't support `depends_on` with simple conditions anymore [[1]](https://docs.docker.com/compose/compose-file/compose-file-v3/).

Reply:
After cleaning and rebuilding images flower does work actually even with broker.